packages/server/graphql/mutations/resetRetroMeetingToGroupStage.ts

```diff
@@ -1,20 +1,36 @@
 import {GraphQLID, GraphQLNonNull} from 'graphql'
 import {SubscriptionChannel} from 'parabol-client/types/constEnums'
 import findStageById from 'parabol-client/utils/meetings/findStageById'
+import {DISCUSS, GROUP, VOTE} from '../../../client/utils/constants'
 import getRethink from '../../database/rethinkDriver'
+import DiscussPhase from '../../database/types/DiscussPhase'
+import DiscussStage from '../../database/types/DiscussStage'
 import GenericMeetingPhase, {
   NewMeetingPhaseTypeEnum
 } from '../../database/types/GenericMeetingPhase'
 import GenericMeetingStage from '../../database/types/GenericMeetingStage'
+import MeetingRetrospective from '../../database/types/MeetingRetrospective'
 import {getUserId} from '../../utils/authorization'
 import publish from '../../utils/publish'
 import standardError from '../../utils/standardError'
-import ResetMeetingToStagePayload from '../types/ResetMeetingToStagePayload'
-import createNewMeetingPhases, {primePhases} from './helpers/createNewMeetingPhases'
+import ResetRetroMeetingToGroupStagePayload from '../types/ResetRetroMeetingToGroupStagePayload'
+import {primePhases} from './helpers/createNewMeetingPhases'
 
-const resetMeetingToStage = {
-  type: GraphQLNonNull(ResetMeetingToStagePayload),
-  description: `Reset meeting to a previously completed stage`,
+const resetMeetingPhase = (phase: GenericMeetingPhase) => {
+  const newStages = phase.stages.map(({id}) => {
+    const newStage =
+      phase.phaseType === DISCUSS
+        ? new DiscussStage({sortOrder: 0, reflectionGroupId: ''})
```

Comment:
if there are 10 discussion topics, this will create 10 dummy stages. that probably won't create an error today, but that means future devs will have to be aware that there could be multiple dummy stages. let's instead just have 1 dummy stage